tema-html-2/temaformsandinputs.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Pizza order</title>
+  </head>
+
+  <body>
+    <form action="https://www.google.com/" method="get">
+      <fieldset>
+        <legend>Pizza order</legend>
+        <div>
+          <label for="yname">Your name</label>
+          <input
+            type="text"
+            id="yname"
+            name="yname"
+            placeholder="Your name"
+            required
+            minlength="3"
+            maxlength="10"
+          />
+        </div>
+
+        <br />
+
+        <div>
+          <label for="email"> Your email</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="Your email"
+          />
```

Comment:
si email-ul ar fi trebuit sa fie required
```suggestion
          <input
            required
            type="email"
            id="email"
            name="email"
            placeholder="Your email"
          />
```